### ISSUE TITLE
video segmentaion?

### ISSUE THREAD

**Issue by antoniaaaaaaaa** [MedicineToken/Medical-SAM2#18]
Can Medsam2 automatically segment organs in endoscopic surgery videos?

**Comment by jiayuanz3** [comment]
yes, but there might be some small changes modified by you, depending on your datasets. in the 3d mode, we consider 3d images as a sequence of 2d images. so similarly, endoscopic surgery videos are automatically a sequence of 2d images from my side. besides, sam2 is designed for videos.
